lib/beaker/hypervisor/docker.rb

```diff
@@ -102,7 +113,7 @@ def provision
         fix_ssh(container) if @options[:provision] == false
 
         @logger.debug("Starting container #{container.id}")
-        container.start({"PublishAllPorts" => true, "Privileged" => true})
```

Comment:
What is the a reason you moved these parameters to `container_opts`? Is there a change in behavior by doing so, or is it just a refactor?